encoding/codecs/h264.md

```diff
@@ -0,0 +1,247 @@
+# Encoding with x264
+
+H.264 has been the de facto standard video codec
+across the internet for the past decade.
+It is widely supported for playback in all modern browsers
+and many hardware devices such as gaming consoles.
+It provides better video quality at smaller file sizes
+compared to its predecessors.
+
+x264 is a mature, free, open-source encoder for the H.264 codec.
+This guide will focus on x264.
+
+## Prerequisites
+
+To get started, you'll need two things:
+
+- A video to encode--for the examples,
+we will pipe in a video from VapourSynth,
+which you should be able to do if you've been following
+the previous sections of this guide
+- The x264 encoder
+
+Here's how we get a copy of the x264 encoder:
+
+### Windows
+
+TODO
+
+### Linux/macOS
+
+Generally, x264 will be available
+through your distribution's package manager.
+Here are a few examples:
+
+#### Ubuntu/Debian
```

Comment:
Formatting-wise, it's likely better to keep this concise in a simple list, e.g.:

```
- **Ubuntu/Debian**: `sudo apt-get install x264`
```

Reply:
Perhaps a nitpick, but does it make sense to still use `apt-get` in guides or tutorials? `apt` was released years ago and has been in Ubuntu and Debian for at least 2 LTS versions.

Reply:
90% of the guides I see still use `apt-get`. But guides like [this](https://itsfoss.com/apt-vs-apt-get-difference/) make me think that's outdated and `apt` is better. Feels like searching for answers on StackOverflow and the first one you find is from 2012.